### DISCUSSION TITLE
Metamong ACT Beta Open ! 사용 팁 공유 드립니다.

### DISCUSSION THREAD

**Discussion by ACT-TJ** [metamong-friends/metamong-friends.github.io#57] [공지사항]
안녕하세요!  TJ 입니다.

Metamong 팀에서 열심히 만든 서비스를 부서 분들께 먼저 소개하게 되어 기쁘고 감사한 마음입니다.

![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/124771422/707a3849-77fa-4202-b1a0-4a92421e6b3c)

미흡한 부분이 있지만, 조금이라도 쾌적한 사용이 될 수 있도록 몇 가지 사용 팁을 전달 드리고자 합니다. 
* 사용경험 개선을 위해 기능의 변경이 발생할 수 있으며, 릴리즈 공지를 통해 공유 드리겠습니다.

1) 공간 활용
. Office 는 public 공간과 회의실 공간으로 나뉘어 각각 다른 기준으로 동작합니다.

[ public 공간 ]

- 근접한 동료들의 Cam, 이모지, 채팅 통한 대화가 가능합니다. 서로 음성 대화는 기본적으로 연결 x
- 동료에게 가까이 접근하여 멈추게 되면, 간이 회의 Dom 이 생성됩니다. 이때 Dom 내부 유저분들만
  Private 음성 대화, 그리드뷰, 화면 공유 등이 가능합니다. Dom 은 동료가 추가되면 확장되며, Private Dom (외부에서 동료가
  Dom 으로 진입 불가) 으로 전환이 가능합니다.
 
 [ Meeting 공간 ] 일반/소규모 회의실, Working zone, Town Hall 

- 동일공간 내 동료들간 음성대화, 그리드뷰, 화면공유 등 협업 기능이 활성화 됩니다.
- Office 위/ 아래쪽 티타임, 소규모 대화를 위한 공간들도 마련되어 있으니 많은 활용 부탁드립니다^^

![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/124771422/9ced2b73-99db-4ffd-aaab-6faca60cc3ed)

2) 성능 관련 ( PC 전력사용, CPU, 그래픽 사용 )

- 좌측 설정메뉴 하단 Frame Rate 를 조절하실 수 있습니다. 랩탑의 사양이나 사용환경 등에 따라 조절 가능하며,
  낮추셔도 급한 화면 이동이나 인터렉션이 많은 상황이 아니라면 사용성에 큰 영향은 없는 상황입니다.

- 내장 + 외장 그래픽 조합의 랩탑의 경우 마우스 깜빡임이 발생할 수 있습니다. 관련 설정 안내 링크 입니다.
https://github.com/metamong-friends/metamong-friends.github.io/discussions/53

3) 기타

- [Lobby, TownHall] 은 스크럼에 속해있지 않거나 다른 스크럼 유저들과 소통할 수 있는 공간으로 가입 시 자동 등록
  되도록 하였습니다. 다른 스크럼 동료들과도 활용해 보시면 좋을 것 같습니다.
  
 - Shift 키를 이용해 보세요. 오픈전까지만 제공될 수 있는 기능들이 숨어 있습니다.^^
  (Shfit + z) (Shift + 숫자)

**** Metamong 사용 경험과 의견이 매우 소중합니다. 본 Discussion 으로 통해 많은 관심과 제보, 의견 부탁 드리겠습니다.
   Usage 에 대한 아이다어, 기능, 개선 아이디어 등 어떤 의견이든 편하게 남겨주시면 감사하겠습니다.**

**Comment by merryblack** [comment]
케이트의 이스터에그 한개 방출!
만약 내 눈앞에 나와 같은 캐릭터의 유저가 나타났다...?!! 이때 **같은 캐릭터가 싫다면!**
Shift + 0 를 잽싸게 눌러보세요😆
